packages/api/tests/utils/TestClient.ts

```diff
@@ -85,7 +85,15 @@ export abstract class TestClient {
     return this._request('updatePreferences', mutations.updatePreferences, { preferences });
   }
 
-  static createModule(module: ModuleInput): Promise<Module> {
+  static createAssignment(assignment: CreateAssignmentInput): Promise<Assignment> {
+    return this._request('createAssignment', mutations.createAssignment, { assignment });
+  }
+
+  static createAssignmentFile(assignmentFile: CreateAssignmentFileInput): Promise<AssignmentFile> {
```

Comment:
As per comment above, let's default each of these parameters with the random functions

Reply:
Will do this in a seperate PR.